### DISCUSSION TITLE
I didn't get student benifits

### DISCUSSION THREAD

**Discussion by kevalmehta17** [community/community#147131] [GitHub Education]
### Hi everyone,

Hello Everyone !!
i applied for github student pack for using github Course but i didn't get any Email!!
It showing me this :- "Congrats! Your request for [@kevalmehta17](https://github.com/kevalmehta17) was approved and your benefits will be available [soon](https://github.com/orgs/community/discussions/111352#user-content-how-long-after-ive-been-approved-will-i-receive-my-academic-benefits)."  ~
but i didn't get anything?? 
Why??

**Comment by davevad93** [comment]
Hi @kevalmehta17 ,

When the benefits become available, the status label will turn purple and change to:

- `Congrats! Your request for @kevalmehta17 was approved and your benefits are now available.`

The waiting time may vary depending on different factors, such as the volume of requests; it usually takes a few days, but sometimes it can take a little longer, even a couple of weeks.

For further reference you can look at these discussions: 
#132790 , #131665 , #132470 , #133531, #134279, #135849, #136887, #138721, #139268, #140287, #145640, #145995

And also the [Student Developer Pack Application FAQs](https://github.com/orgs/community/discussions/111352) :

- [How long after I submit my application will it be approved?](https://github.com/orgs/community/discussions/111352#how-long-after-i-submit-my-application-will-it-be-approved)
- [How long after I’ve been approved will I receive my academic benefits?](https://github.com/orgs/community/discussions/111352#how-long-after-ive-been-approved-will-i-receive-my-academic-benefits)
